{{cookiecutter.github_project_name}}/lib/plugin.js

```diff
@@ -2,6 +2,7 @@ module.exports = [{
     id: '{{ cookiecutter.extension_name }}',
     autoStart: true,
     activate: function(app) {
-       console.log(app.commands);
+      console.log('{{ cookiecutter.extension_name }} extension is active.');
```

Comment:
@danielballan What do you think about changing the log message to:

'Extension {{ cookiecutter.extension_name }} is active.'

Primarily to avoid when folks use extension in the name (i.e. popup-extension) would read 'popup-extension extension is active.'